src/app-layout/visual-refresh-toolbar/skeleton/styles.scss

```diff
@@ -153,11 +153,11 @@
   margin-block-start: awsui.$space-scaled-s;
   margin-block-end: awsui.$space-layout-content-bottom;
   &-disable-paddings {
+    margin-block: 0;
     grid-column: 1 / -1;
     @include desktop-only {
       grid-column: 2 / span 3;
     }
-    margin-block: 0;
   }
```

Comment:
This snippet demonstrates the deprecated pattern very clear. Top level styles should always go first, before any nested rules.